doc/api_assets/style.css

```diff
@@ -93,7 +93,10 @@ em code {
   color: white !important;
   margin: 0 0 1em 0;
   font-family: "Lato", "Lucida Grande", "Lucida Sans Unicode", "Lucida Sans", Verdana, Tahoma, sans-serif;
-  font-weight: 700;
+  white-space: pre-wrap;
+  word-wrap: break-word;
+  padding: 1em;
+  line-height: 1.5;
```

Comment:
I reverted the wrapping changes. Just to clarify, I don't know if the `line-height` is strictly "necessary", I just added it because that was what the `pre` had originally. I don't know if you all think it looks fine without it.

Without `line-height`:
<img width="1143" alt="screen shot 2017-08-14 at 5 18 42 am" src="https://user-images.githubusercontent.com/5678694/29265727-1f1436ea-80b0-11e7-8990-0358673653d1.png">


With `line-height`:
<img width="1142" alt="screen shot 2017-08-14 at 5 18 53 am" src="https://user-images.githubusercontent.com/5678694/29265729-2311f944-80b0-11e7-8728-34f25e871cac.png">